### DISCUSSION TITLE
Multiple lending problems

### DISCUSSION THREAD

**Discussion by bouyeijiang** [rusqlite/rusqlite#1219] [General]
for an example:  

 let mut pre=conn.prepare(command_text)?;
 let rows= pre.query([])?;

// let columns=pre.column_names();//borrow error
//Are there any other alternatives？

let in= =pre.exists([]);//is ok

**Comment by gwenn** [comment]
https://docs.rs/rusqlite/latest/rusqlite/struct.Rows.html#method.as_ref
